Formula/tmux-xpanes.rb

```diff
@@ -0,0 +1,26 @@
+class TmuxXpanes < Formula
+  desc "Ultimate terminal divider powered by tmux"
+  homepage "https://github.com/greymd/tmux-xpanes"
+  url "https://github.com/greymd/tmux-xpanes/archive/v2.2.1.tar.gz"
+  sha256 "b0c537f078e8ba2657da8f66c9ac5b0ed31d558c55bd0fff3b7898791dee1a1e"
+
+  depends_on "tmux" => :recommended
+
+  def install
+    bin.install Dir["bin/*"]
```

Comment:
This should have an `install` script that does something like
```
    system "./install.sh", prefix
```

Reply:
Fixed (b8b2c3c)
The patch version is incremented for making this change by the way.